app/models/message.rb

```diff
@@ -0,0 +1,2 @@
+class Message < ApplicationRecord
```

Comment:
- [ ] Kindly fix the following error that appears when all of the other package error are fixed. You can read more about cors and how to fix it [here](https://medium.com/@chris.lty07/configuring-for-cors-and-cookies-with-react-and-ruby-on-rails-1c30219cc2db).

![image](https://github.com/katarighe/hello-rails-back-end/assets/62290545/31aa175c-4c4a-42cb-8f19-b7f94c666168)

Reply:
✅ It is now working fine.
![image](https://github.com/katarighe/hello-rails-back-end/assets/80690364/bf711897-389f-4301-a217-c275596590eb)

Reply:
✅ It is now working fine for me right now.